cpp/include/cudf/hashing/detail/hash_functions.cuh

```diff
@@ -29,7 +29,7 @@ template <typename T>
 T __device__ inline normalize_nans(T const& key)
 {
   if constexpr (cudf::is_floating_point<T>()) {
-    if (std::isnan(key)) { return std::numeric_limits<T>::quiet_NaN(); }
+    if (std::isnan(key)) { return cuda::std::numeric_limits<T>::quiet_NaN(); }
```

Comment:
This function is device-only. We probably want to consider using `cuda::numeric_limits` and `cuda::isnan` first. If those are unavailable, we can fall back to `cuda::std::isnan` and `cuda::std::numeric_limits`.

Reply:
I looked, but unfortunately there doesn't appear to be a `cuda/limits` header available yet.